app/jobs/release_object_job.rb

```diff
@@ -0,0 +1,117 @@
+class ReleaseObjectJob < GenericJob
+  queue_as :release_object
+
+  attr_reader :manage_release, :pids, :webauth
+  ##
+  # This is a shameless green approach to a job that calls release from dor
+  # services app and then kicks off release WF.
+  # @param [Integer] bulk_action_id GlobalID for a BulkAction object
+  # @param [Hash] params additional parameters that an Argo job may need
+  # @option params [Array] :pids required list of pids
+  # @option params [Hash] :manage_release required Hash of release options
+  # @option manage_release [String] :to required release to target
+  # @option manage_release [String] :who required username of releaser
+  # @option manage_release [String] :what required type of release (self, collection)
+  # @option manage_release [String] :tag required (true, false)
+  # @option webauth [Hash] required for permissions check
+  def perform(bulk_action_id, params)
+    @manage_release = params[:manage_release]
+    @webauth = OpenStruct.new params[:webauth]
+    @pids = params[:pids]
+    File.open(bulk_action.log_name, 'w') do |log|
+      log.puts("#{Time.current} Starting ReleaseObjectJob for BulkAction #{bulk_action_id}")
+      update_druid_count
+
+      pids.each do |current_druid|
+        log.puts("#{Time.current} Beginning ReleaseObjectJob for #{current_druid}")
+        unless can_manage?(current_druid)
+          log.puts("#{Time.current} Not authorized for #{current_druid}")
+          next
+        end
+        log.puts("#{Time.current} Adding release tag for #{manage_release['to']}")
+        begin
+          response = post_to_release(current_druid, params_to_body(manage_release))
+          if response.status == 201
+            log.puts("#{Time.current} Release tag added successfully")
+          else
+            log.puts("#{Time.current} Release tag failed POST #{response.env.url}, status: #{response.status}")
+            bulk_action.increment(:druid_count_fail).save
+            next
+          end
+        rescue Faraday::TimeoutError, Faraday::ConnectionFailed => e
+          log.puts("#{Time.current} Release tag failed POST #{e.class} #{e.message}")
+          bulk_action.increment(:druid_count_fail).save
+          next
+        end
+        log.puts("#{Time.current} Trying to start release workflow")
+        begin
+          response = post_to_release_wf(current_druid)
+          if response.status == 201
+            log.puts("#{Time.current} Workflow creation successful")
+            bulk_action.increment(:druid_count_success).save
+          else
+            log.puts("#{Time.current} Workflow creation failed POST #{response.env.url}, status: #{response.status}")
+            bulk_action.increment(:druid_count_fail).save
+          end
+        rescue Faraday::TimeoutError, Faraday::ConnectionFailed => e
+          log.puts("#{Time.current} Workflow creation failed POST #{e.class} #{e.message}")
+          bulk_action.increment(:druid_count_fail).save
+        end
```

Comment:
So I have in a branch breaking this out even further (release management client class) but wanted to offer a simpler approach in this PR. Just wanted to reduce overall complexity here and leave this less complicated for now. Thoughts?